helm/alfresco-content-services/templates/config-repository-infrastructure.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: infrastructure-repository
+  labels:
+    {{- include "repository.labels" . | nindent 4 }}
+data:
+  repo_svc_name: {{ template "content-services.shortname" . }}-repository
+  repo_svc_port: {{ .Values.repository.service.externalPort | quote }}
```

Comment:
maybe it's an overkill, but we could use the values instead of hardcoding keys here

Reply:
what values?

Reply:
I mean to not hardcode/duplicate `repo_svc_name` and `repo_svc_port` here since you have values for them

Reply:
Ah ok! TBH I didn't even realized I could do that as this kind of goes against the logic I had in mind which is that the configmap is not particularly linked to the share subchart but would be something one can reuse/enrich  to use in another subchart. Actually it's more linked to the fact the parent chart is using the repo component (eventually another subchart), than to share. So it would not make sense the use Share value more than another.